### DISCUSSION TITLE
How to restore deleted plan

### DISCUSSION THREAD

**Discussion by toyoto** [aredden/electron-G14Control#50] [Q&A]
Hello and thanks for your super nice software! The _**Omega Saver plan**_ allows me to record music, because now the laptop is totally silent, thanks to G14Control. But I delete it by error. How to restore it? I tried to install G14control again but without any success. Somebody knows where I can get profile files I could download and import?

**Comment by aredden** [comment]
If you turn off g14controlv2, then go to appdata/roaming/G14ControlV2/G14ControlV2.json and delete that file.. it will delete all of your plans and config stuff.. but after you delete it, open the app again and it will rebuild the original config that comes with the app from a template that is included in the original install.

**Reply by toyoto** [reply]
Thank you so much, it worked !
